atc/metric/metrics.go

```diff
@@ -126,6 +129,8 @@ func (event WorkerContainers) Emit(logger lager.Logger) {
 			Attributes: map[string]string{
 				"worker":   event.WorkerName,
 				"platform": event.Platform,
+				"team":     event.Team,
+				"tags":     strings.Join(event.Tags[:], " "),
```

Comment:
Oh, I didn't know prometheus' text format allowed this (interestingly enough, it allows any utf8! cool! (see [exposition format](https://github.com/prometheus/docs/blob/5b4813eeb364d708278d54ee82cda15d2832b91f/content/docs/instrumenting/exposition_formats.md#comments-help-text-and-type-information))).

I wonder if this would be a problem with the other emitters though 🤔  Do you know if that'd be a problem for the other emitters supported?

Reply:
Thanks for the heads-up. I do a little investigate here.
For `dogstatsd`:
- <https://docs.datadoghq.com/tagging/#defining-tags>
- It support slashes, likes `tags:tag_1/tag_2`

For `InfluxDB`
- <https://docs.influxdata.com/influxdb/v1.7/write_protocols/line_protocol_reference/#special-characters>
- For "tag values", slashes don't need to escapes by a backslash

For `lager` and `newRelic`
- I run a little test, using slashes also works fine

For `riemann`, seems it's using `github.com/golang/protobuf` for serialization.
- In my test, using slashes works fine

So I decide using slashes as tags separator. Wdyt?